tests/AssertTest.php

```diff
@@ -289,7 +290,11 @@ public function getTests()
             array('uuid', array('ff6f8cb0-c57da-51e1-9b21-0800200c9a66'), false),
             array('uuid', array('af6f8cb-c57d-11e1-9b21-0800200c9a66'), false),
             array('uuid', array('3f6f8cb0-c57d-11e1-9b21-0800200c9a6'), false),
-
+            array('throws', array(function() { throw new LogicException('test'); }, 'LogicException'), true),
+            array('throws', array(function() { throw new LogicException('test'); }, 'IllogicException'), false),
+            array('throws', array(function() { throw new Exception('test'); }), true),
+            array('throws', array(function() { trigger_error('test'); }, 'Throwable'), true, false, '7.0.0'),
+            array('throws', array(function() { trigger_error('test'); }, 'Unthrowable'), false, false, '7.0.0'),
```

Comment:
I agree with wath @Nyholm answered you before about it. 
I think `trigger_error` is more accurate, because that's what happens in userland, usually: you trigger errors in php, you don't throw errors.
Errors are either thrown by php (type error, parse error, …), or using `trigger_error`, **not** using `throw new \Error` (which does not exist in php5).